.changeset/shy-mugs-mate.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-analytics-module-ga4': major
```

Comment:
While this plugin is still new, I recommend keeping this on a `0.x.y` version; suggest you swap this from `major` to `minor`.